src/ucp/core/ucp_ep.h

```diff
@@ -822,4 +827,17 @@ void ucp_ep_reqs_purge(ucp_ep_h ucp_ep, ucs_status_t status);
  */
 ucs_status_t ucp_ep_query_sockaddr(ucp_ep_h ucp_ep, ucp_ep_attr_t *attr);
 
+/**
+ * @brief Realloc slow lanes according to current number of lanes.
+ *        Slow lanes number will be (new_num_lanes - MAX_FAST_PATH_LANES).
```

Comment:
IMO it should be done by a caller, or del "slow" from func name

Reply:
what is "done by a caller" mean?

Reply:
new_num_lanes - MAX_FAST_PATH_LANES

Reply:
wouldn't it be more complicated for callers? each caller will have to calculate this value seperatly